src/_how_tos/sw-cli.md

```diff
@@ -68,7 +69,7 @@ For those who are curious, let's look at what's in the
 generated service worker.
 
 ```
-importScripts('sw-lib.v0.0.13.min.js');
+importScripts('workbox-sw.prod.v0.0.2.js');
```

Comment:
Instead of hardcoding `v0.0.2`, can you use `vX.Y.Z`?

If you feel that would be confusing, you could specifically call out that `vX.Y.Z` will in actuality be the latest version number.